### PR TITLE
Homework_4

### PR DESCRIPTION
**Pull request by vadym-yats**

### CODE REVIEW

**Review by katemalash — CHANGES_REQUESTED** [review]
Please improve task 2

File: Yatsentiuk_Vadym/HW4/hw4_2.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+figure = input("What do you choose: rectangle, triangle or circle? ")
+
+if figure == "rectangle":
+    first_side_rectangle = int(input("Enter the first side: "))
+    second_side_rectangle = int(input("Enter the second side: "))
+    print("The area of ​​the rectangle:", str(first_side_rectangle*second_side_rectangle))
+elif figure == "triangle":
+    side_triangle = int(input("Enter the first side: "))
+    height_triangle = int(input("Enter the height triangle: "))
```
[the remark this answers is not in the document]
Comment (vadym-yats) [reply Yatsentiuk_Vadym/HW4/hw4_2.py]: Already fixed, thanks!

File: Yatsentiuk_Vadym/HW4/hw4_3.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+digit = int(input("Enter value: "))
+
+if digit > 0:
+    print("This is a positive number, it's length", len(str(digit)), "characters")
+elif digit < 0:
+    print("This is a negative number, it's length", str(len(str(digit)) -1), "characters")
+else:
+    print("This number is 0, its length is 1 character")
```
[the remark this answers is not in the document]
Comment (vadym-yats) [reply Yatsentiuk_Vadym/HW4/hw4_3.py:8]: Thanks!

File: Yatsentiuk_Vadym/HW4/hw4_5.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+amount = int(input("Enter amount: "))
+
+if amount%200 == 0:
+    print(str(amount//200), "banknotes")
+elif amount%200 != 0 and amount%100 == 0:
```
[the remark this answers is not in the document]
Comment (vadym-yats) [reply Yatsentiuk_Vadym/HW4/hw4_5.py]: She is really not needed, thank you.

File: Yatsentiuk_Vadym/HW4/hw4_6.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+number_of_place = int(input("Enter number: "))
+
+if number_of_place > 36 and number_of_place % 2 == 0 and number_of_place <= 54:
+    print("Your place on top, on the side")
+elif number_of_place > 36 and number_of_place % 2 != 0 and number_of_place <= 54:
+    print("Your place on bottom, on the side")
+elif 1 <= number_of_place <= 36 and number_of_place % 2 != 0:
+    print("Your place on bottom, on the compartment")
+else:
+    print("Your place on top, on the compartment")
```
[the remark this answers is not in the document]
Comment (vadym-yats) [reply Yatsentiuk_Vadym/HW4/hw4_6.py:10]: :)

**Review by katemalash — APPROVED** [review]
Please merge this pull-request

**Pull request merged**